### PR TITLE
Fix microsoft execution

### PR DESCRIPTION
**Pull request by jseremba**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,4 +9,4 @@ test/.project
 *.lock
 *.bak
 .idea/
-*.mws
```
Comment (McDoyen) [inline .gitignore:11]: Missing end of line. Please configure your IDE to do add this.

File: README.md [hunk 2]
```diff
@@ -2,9 +2,11 @@
 
 The BarcodeScanner widget enables PhoneGap native barcode scanning functionality within your Mendix mobile application. This is a widget that will be functional from Mendix 7.13.1.
 
-## Contributing
-
-For more information on contributing to this repository visit [Contributing to a GitHub repository](https://world.mendix.com/display/howto50/Contributing+to+a+GitHub+repository)!
+## Add plugin
+When Publish App for Mobile App Stores, add in the the Custom Phonegap/Cordova configuration the barcode plugin
```
Comment (McDoyen) [inline README.md]: Grammar.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "BarcodeScannerForPhoneGap",
-  "version": "3.1.0",
+  "version": "3.2.1",
```
Comment (McDoyen) [inline package.json:3]: How did you move from v3.1.0 to v3.2.1???? And please check appstore. (Is this a patch or a minor?)

File: src/package.xml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <package xmlns="http://www.mendix.com/package/1.0/">
-    <clientModule name="BarcodeScanner For PhoneGap" version="3.1.0" xmlns="http://www.mendix.com/clientModule/1.0/">
+    <clientModule name="BarcodeScanner For PhoneGap" version="3.2.1" xmlns="http://www.mendix.com/clientModule/1.0/">
```
Comment (McDoyen) [inline src/package.xml:3]: https://semver.org/

**Review by McDoyen — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (jseremba) [reply .gitignore:11]: Added

File: README.md (hunk 2 above)
Comment (jseremba) [reply README.md]: fixed

File: package.json (hunk 3 above)
Comment (jseremba) [reply package.json:3]: There current app store version before the microflow fix is 3.2.0

File: src/package.xml (hunk 4 above)
Comment (jseremba) [reply src/package.xml:3]: Merged branch `feature/offline` that has version 3.2.0

**Pull request closed without merging**